stdlib/2/urllib2.pyi

```diff
@@ -11,8 +13,8 @@ class URLError(IOError):
 
 class HTTPError(URLError, addinfourl):
     code = ...  # type: int
-    headers = ...  # type: Dict[str, str]
-    def __init__(self, url, code, msg, hdrs, fp) -> None: ...
+    headers = ...  # type: Message
+    def __init__(self, url, code: int, msg: str, hdrs: Message, fp: IO) -> None: ...
```

Comment:
Ideally, we would use a protocol for the `fp` pointer as it's only required to have `read()` and `readline()` methods, according to the [implementation of `addbase`](https://github.com/python/cpython/blob/34b7c438b8dc0a1e7e23c9b2d7ce7f8a7c31b4f4/Lib/urllib.py#L955-L970). But for now, I'd be content with just using `Any`. Also applies to a few more functions below.

Reply:
836ca955 is my first stab at protocols; do you prefer that or should I go for `Any`?